packages/components/src/TokenAmountInput/TokenAmountInput.tsx

```diff
@@ -36,3 +43,7 @@ function getCurrencyIdentifier(currency: Token) {
 function makeAmount(value: BN, currency: Token) {
   return new TokenAmount(value, currency);
 }
+
+function renderSelectOnUnpicked() {
```

Comment:
Add renderSelectOnUnpicked to the LiquidityAmountInput, let it be "Select Currency" by default